packages/@sanity/components/src/panes/DefaultPane.js

```diff
@@ -295,41 +297,78 @@ class Pane extends React.Component {
     )
   }
 
+  renderActionMenuItem = item => {
+    const {styles} = this.props
+    if (!item) {
+      return null
+    }
+    const params = item.intent.params
+    const schemaType = S.documentTypeListItem(params.type).spec.schemaType
+    const icon = schemaType.icon
+    return (
+      <IntentLink
+        className={styles.initialValueTemplateDropDownItem}
+        intent="create"
+        params={params}
+      >
+        <div>
+          <div>{item.title}</div>
+          <div className={styles.initialValueTemplateSubtitle}>{params.type}</div>
+        </div>
+        <div className={styles.initialValueTemplateDropDownItemIcon}>{icon()}</div>
```

Comment:
Since icons can be classes as well as stateless function components, it's best to use the `<Icon />` form.